src/ros_can_nodes/include/ros_node_lib/RosNode.hpp

```diff
@@ -0,0 +1,208 @@
+/*
+ * Date Started:
+ * Original Author: Nuno Das Neves
+ * Editors: Yiwei Han
+ * ROS Node Name:
+ * ROS Package: ros_can_nodes
+ * Purpose:
```

Comment:
could you add this please

Reply:
do you mean filling out the ros node name and purpose?

Reply:
Mainly purpose